src/main.rs

```diff
@@ -210,7 +235,15 @@ static MIGRATION_SETTINGS: OnceCell<MigrationSettings> = OnceCell::const_new();
 
 #[tokio::main]
 async fn main() -> CliResult {
-    let cli = Cli::parse();
+    let mut cli = Cli::parse();
+    let base_dir = PathBuf::from(&cli.global_opts.netconfig_base_dir);
+
+    if cli.global_opts.netconfig_path.is_empty() {
+        cli.global_opts.netconfig_path = base_dir.join("config").to_str().unwrap().to_string();
+    }
+    if cli.global_opts.netconfig_dhcp_path.is_empty() {
+        cli.global_opts.netconfig_dhcp_path = base_dir.join("dhcp").to_str().unwrap().to_string();
+    }
```

Comment:
I would say maybe using PathBuf in the args directly could simplify things a bit. I've had a go at it https://github.com/jcronenberg/wicked2nm/commit/f40bb6362eca082c2d93c7a48a9640bab98f3845
Unfortunately one side effect seems to be for whatever reason setting the env vars like this `W2NM_NETCONFIG_PATH=` seems to run into issues with the `Option` because it technically is set. I think this shouldn't be an issue and just required some small changes in the test script. WDYT?

Reply:
yes, lets use PathBuf